### ISSUE TITLE
NTR, pulmo: cobblestoning

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6392]
**Preferred term label:** Presence of pleural cobblestoning

**Synonyms**

**Definition (free text, please give PubMed ID)**
Demonstrating irregular bumps surrounded by depressed lines in open lung biopsy. Caused by underlying scarring causing retraction of the interlobular septa where they insert on the pleura. 

ISBN 978-1-4511-7643-8
Churg, Andrew, and Churg. Atlas of Interstitial Lung Disease Pathology : Pathology with High Resolution CT Correlations, Wolters Kluwer, 2013. ProQuest Ebook Central,

**Parent term (use hpo.jax.org/app)**
Interstitial pulmonary abnormality HP:0006530

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Usual interstitial pneumonia

**Your nano-attribution (ORCID)**
LMU:chrapp, @kknoflach, @mgriese

**Comment by pnrobinson** [comment]
I see the HPO has
https://hpo.jax.org/app/browse/term/HP:0031631 (Subpleural honeycombing) -- this seems to be a synonym?
The HPO also has
Honeycomb lung HP:0025175 -- I think that cobblestoning is a component of this. 
Let's discuss at the next meeting.

**Comment by pnrobinson** [comment]
adding term as recommended, thanks
